xml/System.Globalization/CultureInfo.xml

```diff
@@ -1208,7 +1208,7 @@ If `name` is <xref:System.String.Empty?displayProperty=nameWithType>, the constr
   
      For more information, see the "Culture and task-based asynchronous operations" section in the <xref:System.Globalization.CultureInfo> topic.  
   
--   By calling the Windows `GetUserDefaultLocaleName` function.  
+-   By calling the `GetUserDefaultLocaleName` function on Windows or the `setlocale` function with category `LC_MESSAGES` on Unix-like systems.  
```

Comment:
on Linux we call ICU API uloc_getDefault 

https://github.com/dotnet/runtime/blob/f1de95ede6792c5a3106ea26f2faef42ad29820b/src/libraries/Native/Unix/System.Globalization.Native/pal_locale.c#L132

the setLocale used in old versions but not used anymore starting from 5.0.

Reply:
> on Linux we call ICU API uloc_getDefault

Indeed.  And `uloc_getDefault` calls `setlocale` (by the call sequence I documented in #4254.  Are you arguing we should document `uloc_getDefault` exclusively?

Reply:
Yes would be good to document the actual call we use because if the ICU decided to change the way picking up the default locale that will make our doc invalid. we don't have to document the ICU API ourselves. make sense?

Reply:
Sure, assuming you want ICU to be part of the documented API, rather than an implementation detail.  Also, I suspect many more developers are familiar with how `setlocale` works than `uloc_getDefault`.  Would it be acceptable to note that `uloc_getDefault` currently uses `setlocale`?

Reply:
> Would it be acceptable to note that uloc_getDefault currently uses setlocale?

Sure, I don't mind to phrase it this way.

Reply:
Great!  I've updated the PR to document `uloc_getDefault` and mention `setlocale`.  Let me know what you think.